examples/contour/03-envoy-deployment.yaml

```diff
@@ -0,0 +1,154 @@
+---
```

Comment:
Only downside to putting this into `examples/contour` is that now `kubectl apply -f examples/contour` (which I use somewhat frequently) will create *both* a DaemonSet and a Deployment for Envoy. But I guess that's preferable to creating a full second copy of the directory.

Reply:
Yeah I do the same. =)

I had the deployment.yaml file outside of that directory but had troubles getting it worked in. Maybe I can try that path again to keep things separate.

Reply:
I moved it out into it's own dir so should keep the old way intact.